db/icat.go

```diff
@@ -21,6 +23,10 @@ func NewICAT(db DatabaseAccessor, userSuffix, zone string, rootResourceNames []s
 	return &ICATDatabase{db: db, userSuffix: userSuffix, zone: zone, rootResourceNames: rootResourceNames}
 }
 
+func (i *ICATDatabase) FixUsername(username string) string {
+	return util.FixUsername(username, &config.Config{UserSuffix: i.userSuffix})
+}
```

Comment:
I added this here because it makes it easier to call this from some of the handler functions. Would probably be better to pass around the actual `Config` value more aggressively, but that can come later.